src/patternfly/components/Avatar/examples/Avatar.md

```diff
@@ -37,3 +57,7 @@ The avatar component provides a default SVG icon. If an image is used it should
 | `.pf-c-avatar` | `<img>` |  Initiates an avatar image. **Required** |
 | `.pf-m-light` | `.pf-c-avatar` | Modifies an avatar for use against a light background. |
 | `.pf-m-dark` | `.pf-c-avatar` | Modifies an avatar for use against a dark background. |
+| `.pf-m-sm` | `.pf-c-avatar` | Modifies an avatar to be small. |
+| `.pf-m-md` | `.pf-c-avatar` | Modifies an avatar to be medium. |
+| `.pf-m-lg` | `.pf-c-avatar` | Modifies an avatar to be large. **Note:** This is the default size. |
+| `.pf-m-xl` | `.pf-c-avatar` | Modifies an avatar to be extra large. |
```

Comment:
Should we include "at the optional breakpoint" since we're generating all those classes?

Reply:
yep! updated with your new link.